### DISCUSSION TITLE
npm run deploy gives ENOENT: no such file or directory

### DISCUSSION THREAD

**Discussion by eli-blanc** [community/community#31156] [Other Feature Feedback, Questions, & Ideas]
Hi, I'm trying to deploy a React app with a node.js backend for the first time on GitHub. 
This is my folder structure:
![folderStructure](https://user-images.githubusercontent.com/112291094/187092853-65de87dc-92c1-4b05-a688-7b53e827b507.png)
The server folder contains a index.js file. The client folder contains the front-end stuff.

Here's the content of the package.json file:
{
  "name": "beu-gaspesien",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "start": "node server/index.js",
    "build": "cd client && npm install && npm run build",
    "predeploy": "npm run build",
    "deploy": "gh-pages -d build"
  },
  "keywords": [],
  "author": "",
  "license": "ISC",
  "dependencies": {
    "bcryptjs": "^2.4.3",
    "exists": "^1.0.1",
    "express": "^4.18.1",
    "missing": "^0.0.1"
  },
  "engines": {
    "node": "18.7.0"
  },
  "devDependencies": {
    "gh-pages": "^4.0.0"
  },
  "homepage": "http://eli-blanc.github.io/beu-gaspesien"
}

When I try to deploy with "npm run deploy", I get 

> The project was built assuming it is hosted at /.
You can control this with the homepage field in your package.json.

The build folder is ready to be deployed.
You may serve it with a static server:

  serve -s build

Find out more about deployment here:

  https://cra.link/deployment

> beu-gaspesien@1.0.0 deploy
> gh-pages -d build

ENOENT: no such file or directory, stat 'D:\beu-gaspesien\build'

I see a build folder inside the client folder, but not at the root. 

Any idea what's wrong? Please let me know if you need more info.
Thanks!

**Comment by eli-blanc** [comment]
Also, please let me know if I used the wrong category, it's my first time posting a question here.

**Reply by queenofcorgis** [reply]
Welcome to Community on Discussions @eli-blanc 🎉 

The Issues Category is specifically for the GitHub products: Issues and Discussions. I've moved yours over so not worry, appreciate you posting this 😄

**Comment by DevRubio** [comment]
lograste solucionar el error?

**Reply by shahid099** [reply]
I have also same problem

**Comment by fabiopsd** [comment]
Same probelm and i still didn't find a way to solve it? anyone who wanna help us?

**Reply by vlatcata** [reply]
I've been on this issue all day, have not found a fix...

**Reply by Lecon-a** [reply]
@Vlatcata, I am having a similar issue now. Any help? Oh, I just fix it now! Thanks!

**Reply by Yousef0sama** [reply]
@Lecon-a, how did you fix it?
